### ISSUE TITLE
Doxygen to DocBook

### ISSUE THREAD

**Issue by drogers3261** [doxygen/doxygen#6444]
Using the following header doxygen generates properly puts the details in PDF, XML and HTML files but not in DocBook xml file:
```
/*--------------------------------------------------------------------------*/
/**
 * @brief
 *      Validate command line string for correct command parameters
 *
 * @param  SeqCmd_T* seqCmd    - Sequence command object to evaluate
 *
 * @return
 *      bool        - true = validated, false = non validated
 *
 * Details
 *      SeqCmd_T->paramStr.tokens[0] = SignalName, "Slot1"
 *      SeqCmd_T->paramStr.tokens[1] = Dwell, "1"
 *
 */
/*--------------------------------------------------------------------------*/
```

**Comment by albert-github** [comment]
I've just pushed a proposed patch with a new implementation of the docbook generator, pull request #6487

**Comment by albert-github** [comment]
Code has been integrated in master on github.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
